test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua

```diff
@@ -0,0 +1,140 @@
+  }},
+  [011] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateApp, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.exitApp,       hLvl = "NONE",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }},
+  [012] = { t = "WEB_VIEW", m = true,  s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateHMI, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.exitApp,       hLvl = "NONE",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }},
+  [013] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateHMI, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.exitApp,       hLvl = "NONE",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }},
+  [014] = { t = "WEB_VIEW", m = true,  s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateApp, hLvl = "LIMITED",    aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
+  }},
+  [015] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateApp, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }},
+  [016] = { t = "WEB_VIEW", m = true,  s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateHMI, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.activateHMI,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
+  }},
+  [017] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateHMI, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.activateHMI,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }}
+}
+
+--[[ Local Functions ]]
+local function doAction(pSS)
```

Comment:
@aderiabin `pSS` not obvious parameter name

Reply:
@LitvinenkoIra Done (see 77f3211)